docs/api_walkthrough.rst

```diff
@@ -188,68 +189,68 @@ Connecting to an already existing token network is quite simple. All that is nee
 
 .. http:example:: curl wget httpie python-requests
 
-   PUT /api/1/connections/0xc9d55C7bbd80C0c2AEd865e9CA13D015096ce671 HTTP/1.1
-   Host: localhost:5001
-   Content-Type: application/json
+    PUT /api/1/connections/0x0f114A1E9Db192502E7856309cc899952b3db1ED HTTP/1.1
+    Host: localhost:5001
+    Content-Type: application/json
 
-   {
-       "funds": 2000
-   }
+    {
+        "funds": 2000
+    }
 
-This will automatically connect to and open channels with three random peers in the token network, with 20% of the funds deposited to each channel. Furthermore it will leave 40% of the funds initially unassigned. This will allow new nodes joining the network to open bi-directionally funded payment channels with this node in the same way that it just opened channels with random nodes already in the network. The default behaviour of opening three channels and leaving 40% of the tokens for new nodes to connect with, can be changed by adding ``"initial_channel_target": 3`` and ``"joinable_funds_target": 0.4`` to the payload and adjusting the default value.
+This automatically opens channels with three random peers in the token network. With 20% of the funds deposited to each channel. Furthermore it leaves 40% of the funds initially unassigned. This allows new nodes joining the network to open payment channels with this node in the same way that it just opened channels with random nodes in the network.
```

Comment:
The ''With ... each channel" part is not a whole sentence, let's leave the comma in front of it.

Reply:
You're right. I am trying to use shorter sentences in general, but it doesn't always work out too well. Thanks for pointing it out. Will fix.